etc/env/example.env

```diff
@@ -113,22 +113,22 @@ TRAEFIK_SOCKET=unix:///var/run/docker.sock
 MINIO_VERSION=latest
 MINIO_UI_PORT=9090
 MINIO_PORT=9000
-MINIO_PUBLIC_URL=https://${CANDIG_DOMAIN}:${MINIO_PORT}
+MINIO_PUBLIC_URL=http://${CANDIG_DOMAIN}:${MINIO_PORT}
```

Comment:
Is this change something that we want to make standard? Will it affect local deployments?